cloudevents-sdk-warp/src/lib.rs

```diff
@@ -0,0 +1,254 @@
+#[macro_use]
+mod headers;
+
+mod server_request;
+mod server_response;
+
+pub mod reply {
+    use crate::server_response::event_to_response;
+
+    use cloudevents::Event;
+    use http::StatusCode;
+    use warp::reply::Response;
+
+    ///
+    /// # Serializes `CE` as a http response
+    ///
+    /// ```
+    /// use cloudevents_sdk_warp::{filter, reply};
+    /// use warp::Filter;
+    /// use warp::Reply;
+    ///
+    /// let routes = warp::any()
+    ///    .and(filter::to_event())
```

Comment:
I think you should add this sample at the root doc of this crate (like https://github.com/cloudevents/sdk-rust/blob/master/cloudevents-sdk-actix-web/src/lib.rs#L1).

Also, in this particular example, you should not use the filter, because you're documenting the `reply`. More than using the filter, just create a new event with `Event::default()`